packages/app/src/cli/utilities/developer-platform-client/shopify-developers-client.ts

```diff
@@ -134,30 +163,57 @@ export class ShopifyDevelopersClient implements DeveloperPlatformClient {
   }
 
   async organizations(): Promise<Organization[]> {
-    return [ORG1]
+    const organizationsResult = await businessPlatformRequest<OrganizationsQuerySchema>(
+      OrganizationsQuery,
+      await this.businessPlatformToken(),
+    )
+    return organizationsResult.currentUserAccount.organizations.nodes.map((org) => ({
+      id: Buffer.from(org.id, 'base64').toString('ascii').match(/\d+$/)![0],
```

Comment:
Context: Business Platform returns the ID as a base64-encoded gid. So we base64-decode, then pull out the numeric portion at the end, which is what we'll use for calls to the Shopify Developers API.

Reply:
I think I'd just add the above as a comment. So someone can see at a glance what these three functions calls add up to